ipywidgets/widgets/tests/test_interaction.py

```diff
@@ -637,31 +637,14 @@ def test_multiple_selection():
 
     # change value
     w.value = w.value + (2,)
-    check_widget(w, value=(1, 2), selected_labels=(1, 2))
-
-    # change value name
-    w.selected_labels = (1,)
-    check_widget(w, value=(1,))
-
-    # don't accept random other names when no options
-    with nt.assert_raises(KeyError):
-        w.selected_labels = (3,)
-    check_widget(w, value=(1,))
-
-    # don't accept selected_label (from superclass)
-    with nt.assert_raises(AttributeError):
-        w.selected_label = 3
-
-    # don't return selected_label (from superclass)
-    with nt.assert_raises(AttributeError):
-        print(w.selected_label)
+    check_widget(w, value=(1, 2))
```

Comment:
It's not clear to me why you removed these tests.  Shouldn't these errors still be thrown?